README.md

```diff
@@ -1,28 +1,31 @@
 ## Udf filesystem golang library
+
 - Non-optimized
-- Some functioal is broken
+- Some features may be broken
 - `recovery()` style error handling interface
-- Work only with certain iso's
+- Tested only with certain ISOs (e.g. Windows ISOs)
 
-It's all because I has reached requried functional for me.
+It's all because I has reached required functionality for me.
 
 ## Example
+
 ```go
 package main
 
 import (
 	"fmt"
 	"os"
-	"github.com/mogaika/udf"
+	"github.com/saidelike/udf"
```

Comment:
Can we keep the original import structure?

Reply:
Indeed